src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/SingleEventRegistrationEntry.component.js

```diff
@@ -0,0 +1,32 @@
+// @flow
```

Comment:
Same as before this file isnt a complete rewrite but only a copied version of the file `Pages/NewEvent/SelectionsComplete/SelectionsComplete.component.js`